### ISSUE TITLE
I'm pretty sure this isn't actually the Solid Gold version

### ISSUE THREAD

**Issue by eriktorbjorn** [the-infocom-files/leathergoddesses-invclues-old#3]
The only difference between this repository and the 'leathergoddesses' one is that this one uses fixed font when displaying the word puzzle on the scrap of paper, so it's probably not really the Solid Gold version. I think Zarf simply mislabelled this, and that it's the leathergoddesses-invclues-rnohyphen.zip one that's the real one.

It seems more likely that this was meant as a Macintosh-specific version of the game. Judging by Zarf's archive, there were such versions of some other games but apparently not of this one? I'm not sure, though. I _do_ have an original Mac disk, but even assuming it's still readable, and even assuming that my old Mac still works, and even assuming I could figure out a way to transfer the file off it, it's the Solid Gold version.

(I did point this out to Zarf, but I guess he hasn't had the time to look into it. He mentioned being busy with NarraScope and moving plans.)

**Comment by AlexProudfoot** [comment]
Hmmm, invclues-r4 compiles to z3 so it IS the regular version. The invclues-rnohyphen version compiles to z5 and definately has invisiclues. It's also the same exact version as the one on Historical Sources. Problem solved.

**Comment by eriktorbjorn** [comment]
That's what I thought, though I don't remember now how closely I actually checked it. So I guess invclues-rnohyphen should be the baseline for leathergoddesses-invclues, and invclues-r4 should be the baseline for leathergoddesses.

**Comment by AlexProudfoot** [comment]
I didn’t quite understand your initial comment about fixed font and didn’t compare the sources to clarify. If we’re talking about a z3 game, how is it able to change the font?

**Comment by eriktorbjorn** [comment]
According to the Z-Machine standards document, the ability to print text in a fixed font was added in version 3 of the Z-Machine, and it's done by setting a bit in the header. I guess this was at least partly because the Macintosh versions of their z3 games used a proportional font by default. (Their z4 and z5 interpreter used only a fixed font, while their z6 interpreter went back to using a proportional font again.)

When looking at only the .zil files, invclues-r4 has a c1.zil that r59 doesn't have, but since it's identical to x1.zil (which both have), I'm discounting that. Other than that, the _only_ difference I can find is in phobos.zil, where r59 defines SCRAP-OF-PAPER like this:

```
<OBJECT SCRAP-OF-PAPER
        (IN OTHER-CELL)
        (DESC "scrap of paper")
        (FDESC
"A crumpled paper lies discarded in the corner. There seems to be some
writing on it.")
        (SYNONYM SCRAP PAPER)
        (ADJECTIVE DISCARDED CRUMPLED)
        (FLAGS TAKEBIT BURNBIT READBIT)
        (SIZE 2)
        (TEXT
"There's a seemingly meaningless matrix of letters on the paper:|
   HESOHREBBUR|
   ILSSSIPNGEF|
   RGIUGHTHDEN|
   SNKOOBENOHP|
   FALYTMERATP|
   SHEADLIGHTO|
   SLLABNOTTOC"
;"after removing all the letters from all the words on the parts list, the
remaining letters spell HISSING FRIGHTENS FLY TRAPS")>
```

While invclues-r4 defines it like this:

```
<OBJECT SCRAP-OF-PAPER
        (IN OTHER-CELL)
        (DESC "scrap of paper")
        (FDESC
"A crumpled paper lies discarded in the corner. There seems to be some
writing on it.")
        (SYNONYM SCRAP PAPER)
        (ADJECTIVE DISCARDED CRUMPLED)
        (FLAGS TAKEBIT BURNBIT READBIT)
        (SIZE 2)
        (ACTION SCRAP-OF-PAPER-F)>

<ROUTINE SCRAP-OF-PAPER-F ()
         <COND (<VERB? READ>
                <TELL
"There's a seemingly meaningless matrix of letters on the paper:" CR>
                <PUT 0 8 <BOR <GET 0 8> 2>> ;"turns on fixed spacing for Mac"
                <TELL
;"after removing all the letters from all the words on the parts list, the
remaining letters spell HISSING FRIGHTENS FLY TRAPS"
"   HESOHREBBUR|
   ILSSSIPNGEF|
   RGIUGHTHDEN|
   SNKOOBENOHP|
   FALYTMERATP|
   SHEADLIGHTO|
   SLLABNOTTOC">
                <PUT 0 8 <BAND <GET 0 8> -3>> ;"turns off fixed spacing"
                <RTRUE>)>>
```

In at least some of the games, there are helper routines called FIXED-FONT-ON and FIXED-FONT-OFF to handle the flag.

**Comment by AlexProudfoot** [comment]
Yes, the historical sources repository makes this code the “final revision” but x1.zip hasn’t been updated so we don’t have a unique release number for it. I guess Zarf should have named it leathergoddesses-rlater.

**Comment by eriktorbjorn** [comment]
Just a heads-up that Zarf has updated his files so that the LGOP source code is now named:

leathergoddesses-r59.zip
leathergoddesses-mac-r59.zip
leathergoddesses-invclues-r4.zip

He's made some other updates as well, but I think most of them are just in the folder name the ZIP files unpack to. I think the only files that have been added since then are zillib.zip and zil.zip.

**Comment by AlexProudfoot** [comment]
Done. See https://github.com/the-infocom-files/leathergoddesses-invclues/releases and let me know if you see any other problems. I’ve just done the minimum changes necessary to build at the moment. I still need to remove unnecessary statements in the main file.

**Comment by eriktorbjorn** [comment]
I was able to compile the new repository without problems, and it's using the expected "develop" branch. I assume the main file may possibly be changing name in the future (since the generated game file is currently called x1.z5) but that shouldn't stop me from submitting a spelling fixes pull request later. Thanks!
